### DISCUSSION TITLE
Confusing paths to me

### DISCUSSION THREAD

**Discussion by SpyrosPsarras** [LadyElyssa/LadyElyssaTacoTrails#18] [General]
Hi,

I am enjoying your markers but I cannot understand what am I supposed to do when I see numbers on the paths. 

I am posting a picture where I see an alternative numbered "4" route but I cannot understand when to follow that number. I was following the main marker path and these numbers where keep coming.

It seems that removing the number from the list, it removes only the number and not the trail. 
![image](https://github.com/LadyElyssa/LadyElyssaTacoTrails/assets/13405649/6125cd24-c157-4734-9723-049e3df1c148)

**Comment by LadyElyssa** [comment]
Thank you for your feedback. What is it about the numbered paths that you find confusing? Is it because they increment as you go through the map? Is it because there is a single trail, that is numbered as 4, that you think would be better numbered as 1 to represent how many paths go from that waypoint?

At that point in Auric Basin the main path goes around to the left, up to the Hero Point and then drops down to the Vinetooth Den to get the POI. Once there it puts a message on screen telling you which Waypoint to return to, as I am sure you are familiar with by now and also copies the chat code to the clipboard so that you can post this in to one of your chat channels to make finding it easier. 

The numbers on the trail are for when you return back to a waypoint, so that you can follow one of the numbered trails. After some further thought I can see why having the number 4 where there is only a single trail might be confusing. In this case the number 4 represents the 4th part of the trail.

**Reply by SpyrosPsarras** [reply]
Thanks for the reply.

It might been better if the colors where different. The thing is that I may not be doing one map in a single sit, so that means that I will have to remember which number was the last one I used. 

I will come back with a better proposal next time. As a user though, I am letting you know that I have tried many (all?) the taco modules and i find this one the most promising in the future but a bit confusing.

**Reply by LadyElyssa** [reply]
Thank you for the follow up and for your initial suggestion of using different coloured routes, which on some maps would certainly have been an option. The main downside to this would be for those with visual impairments that have difficulty seeing some shades or colours. So for some trails, like those for hero points, I use different trail markers to help with this.

I have been pondering your initial point about the trail numbers, which as you had noticed only toggle the trail number on and off and don't affect the trail due to it effectively being one long trail. While it would be a lot of work, this could be broken down in seperate trails. Which I can only presume is perhaps what you were originally inferring and that you could then toggle on / off as you complete each segment and also offer you a means to effectively track your progress.

Thank you for the feedback and Look forward to hearing any proposals for potential improvements.

**Comment by LadyElyssa** [comment]
I am about to upload a quick update which should be available through BlishHUD within 30 minutes where you can take a look at the Map Completion trails within the Silverwastes. Please do let me know if this is the kind of thing you were hoping for so I can go see about redoing the rest of the map completion trails to work the same way.

Addendum: Have also done this for the HoT maps now too and will work my way through some of the maps for living world along with those from the other expacs before eventually returning to do the same for all of the core maps.

Edit: I have so far applied this to the living world maps and all the expac maps.
